src/chess/uciloop.cc

```diff
@@ -131,8 +133,13 @@ void UciLoop::RunLoop() {
     LOGFILE << ">> " << line;
     try {
       auto command = ParseCommand(line);
-      // Ignore empty line.
-      if (command.first.empty()) continue;
+      // Unknown UCI command, check non standard.
+      if (command.first.empty()) {
+        if (!CmdNonStandardUciCommand(line)) {
```

Comment:
Does it work this way?.. We should get there only when command is empty, but in this case CmdNonStandardUciCommand() will have nothing to match against.

I suggest we either call `CmdNonStandardUciCommand()` from within DispatchCommand() (and in this case we need to add "get" command into `kKnownCommands` like it's done for non-standard "xyzzy" command), or

call "TryHandleNonStandardCommand()" before calling ParseCommand() (and skip parsing if it's successful). That doesn't look very nice though.